src/CMakeLists.txt

```
@@ -2,6 +2,9 @@
 
 # Create a list of the files to be preprocessed
 set(fppFiles
+    stdlib_bitset_64.fypp
+    stdlib_bitset_large.fypp
```

I suggest to call these 2 files as follows:
```suggestion
    stdlib_bitsets_64.fypp
    stdlib_bitsets_large.fypp
```

---

What are your reasons for changing the file names? Changing them affects five files, the two files whose names are changed, them CMakeLists.txt file, the Makefile.manual file, and the markdown document.

---

So they would have the same prefix ("stdlib_bitsets") as the file stdlib_bitsets.fypp. I always find such things easier to edit (e.g. I don't need to think if a "s" is needed or not). If others have no objections against the current names, it is also fine for me.